### PR TITLE
Initial work to use GSSCredential interface

### PR DESCRIPTION
**Pull request by worksofliam**
Signed-off-by: Liam Allan <mrliamallan@live.co.uk>

### CODE REVIEW

File: src/main/java/com/ibm/as400/access/AS400.java [hunk 1]
```diff
@@ -3934,7 +3936,7 @@ static Object getGSSManager()
      Sets the GSS credential for this object.  Using this method will set the authentication scheme to {@link #AUTHENTICATION_SCHEME_GSS_TOKEN AUTHENTICATION_SCHEME_GSS_TOKEN}.  Only one authentication means (Kerberos ticket, profile token, identity token, or password) can be used at a single time.  Using this method will clear any previously set authentication information.
      @param  gssCredential  The GSS credential object.  The object's type must be org.ietf.jgss.GSSCredential, the object is set to type Object only to avoid a JDK release dependency.
```
Comment (ThePrez) [inline src/main/java/com/ibm/as400/access/AS400.java:3937]: can clean up this comment now

**Comment by worksofliam** [comment]
Regarding this line of code: https://github.com/worksofliam/JTOpen/blob/fix/50_gsscredential/src/main/java/com/ibm/as400/access/AS400ImplProxy.java#L284

> for now, just change the setGSSCredential signature, but dang, we should open an issue to revisit this

**Review by ThePrez — CHANGES_REQUESTED** [review]

File: src/main/java/com/ibm/as400/access/AS400.java (hunk 1 above)
Comment (ThePrez) [reply src/main/java/com/ibm/as400/access/AS400.java]: Cleaned up.

**Comment by ThePrez** [comment]
@worksofliam , the changes LGTM! 
I see we should also fix up `setGSSManager()`. Should you handle here or should we do a separate issue/pr?

**Review by ThePrez — APPROVED** [review]

**Pull request merged**